### PR TITLE
Medibase Medicines Integration

### PR DESCRIPTION
**Pull request by rithviknishad**
## Proposed Changes

- Adds Medibase Medicines static data
- Migrations to load it to MedibaseMedicine table
- In-memory table to search MedibaseMedicines based on name/generic/company.
- Old medicines are moved to `medicine_old`
- Updated discharge summary to show appropriate medicine.

Eg. discharge summary: [84115ae4-9636-4aac-948a-fb10445c89a41686141886.pdf](https://github.com/coronasafe/care/files/11777809/84115ae4-9636-4aac-948a-fb10445c89a41686141886.pdf)

## Migrations:

1. Rename field `medicine` to `medicine_old`
2. Alter field: `medicine_old` to be nullable and added field `medicine` as nullable not blank FK to `MedibaseMedicine`
3. Custom Migration that populates `MedibaseMedicine` table from `static_data`

## Merge Checklist

- [ ] Tests added/fixed
- [x] Update docs in `/docs`
- [x] Linting Complete
- [x] Any other necessary step

_*Only PR's with test cases included and passing lint and test pipelines will be reviewed*_

@coronasafe/care-backend-maintainers @coronasafe/care-backend-admins

### CODE REVIEW

File: care/facility/models/prescription.py [hunk 1]
```diff
@@ -37,6 +37,60 @@ def generate_choices(enum_class):
...
+    )
+    cims_class = models.CharField(
+        max_length=255,
+        blank=True,
+        null=True,
+    )
+    atc_classification = models.TextField(
+        blank=True,
+        null=True,
+    )
```
Comment (sainak) [inline care/facility/models/prescription.py:88]: dont indent if not necessary

File: care/facility/models/prescription.py [hunk 2]
```diff
@@ -37,6 +37,60 @@ def generate_choices(enum_class):
...
+
+    def __str__(self):
+        return self.name + " - " + self.type
```
Comment (sainak) [inline care/facility/models/prescription.py:91]: fstrings are much better

**Comment by rithviknishad** [comment]
Added migrations from fresh + used `name` as unique instead of `medibase_id` since that was a Mongodb thing.

image

**Review by sainak — APPROVED** [review]

**Review by sainak — APPROVED** [review]

**Comment by rithviknishad** [comment]
image

**Review by sainak — APPROVED** [review]

**Review by vigneshhari — APPROVED** [review]

**Pull request merged**